src/libinjection_xss.c

```diff
@@ -574,7 +573,7 @@ static int cstrcasecmp_with_null(const char *a, const char *b, size_t n) {
  *
  * return 1 if match / starts with
  * return 0 if not
- */
+ */                            /*const char* prefix, const char *src, size_t n*/
```

Comment:
```suggestion
 */
 /*const char* prefix, const char *src, size_t n*/
```
But I'm not sure this comment is necessary here.